app/src/vanilla/java/cm/aptoide/pt/app/view/AppViewPresenter.java

```diff
@@ -612,6 +600,22 @@ private void handleClickOnRetry() {
             .map(__ -> appViewModel));
   }
 
+  @NonNull private Observable<Experiment> handleRecommendsExperiment(AppViewViewModel appViewModel,
+      Account account) {
+    return appViewManager.getShareDialogExperiment()
+        .observeOn(viewScheduler)
+        .doOnNext(
+            experiment -> showRecommendsDialog(account.isLoggedIn(), appViewModel.getPackageName(),
+                experiment))
+        .doOnError(throwable -> {
```

Comment:
There is no need for this to be sent as an error, we could just send the "default" error empty experience from the AppViewManager